docs/localdev.md

```diff
@@ -93,6 +93,9 @@ This section walks you through the deployment of the external-dns-operator in a
 - [Go](https://golang.org/doc/install#install) and [Docker](https://docs.docker.com/engine/install/) are installed.
 - [Kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation) is installed and a local Kubernetes [cluster](https://kind.sigs.k8s.io/docs/user/quick-start/#creating-a-cluster) is created.
 - [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) is installed to access the above created cluster.
+- Ensure the above created cluster is accessible by any `kubectl` command. By default, it looks for the kubeconfig file which is usually present at `~/.kube/config` when a cluster is created. 
+- If you want `kubectl` to point to a different cluster, set the `$KUBECONFIG` env variable pointing to that cluster's config file.
+`export KUBECONFIG=</path/to/the/kubeconfigfile>`
```

Comment:
Maybe explain how to get the kubeconfig in kind
i.e execute this command (I used podman replace it with docker)
`sudo podman exec  $(sudo podman ps -a | grep docker.io/kindest | awk '{print $1}')  "/bin/sh" "-c" "cat /etc/kubernetes/controller-manager.conf"`

Reply:
Okay, I would add it. But isn't that enough if we mention that the user has to make sure the cluster is accessible by the "kubectl" ?